main/main.go

```diff
@@ -39,8 +38,16 @@ func main() {
 	if err := fs.Parse(os.Args[1:]); err != nil {
 		panic(fmt.Errorf("couldn't parse flags: %w", err))
 	}
-	handleInfoDisplayFlags(fs)
-	v, err := config.BuildViper(fs, os.Args[1:])
+	// Having the version flag set, we display the version then exit
```

Comment:
```suggestion
	// If the version flag set, display the version then exit
```
Prefer avoiding "we". Also think this phrasing is less awkward.